config/karma.saucelabs.js

```diff
@@ -86,11 +85,11 @@ function appiumLauncher(
  */
 const sauceLabsBrowsers = {
   // Desktop
-  Chrome_Windows: seleniumLauncher('chrome', 'Windows 10', 'latest'),
-  Firefox_Windows: seleniumLauncher('firefox', 'Windows 10', 'latest'),
-  Safari_macOS: seleniumLauncher('safari', 'macOS 10.13', 'latest'),
-  Edge_Windows: seleniumLauncher('MicrosoftEdge', 'Windows 10', 'latest'),
-  IE_Windows: seleniumLauncher('internet explorer', 'Windows 10', 'latest')
+  Chrome_Windows: seleniumLauncher('chrome', 'Windows 10', '66.0'),
+  Firefox_Windows: seleniumLauncher('firefox', 'Windows 10', '60.0'),
+  Safari_macOS: seleniumLauncher('safari', 'macOS 10.13', '11.0'),
+  Edge_Windows: seleniumLauncher('MicrosoftEdge', 'Windows 10', '17.17134'),
+  IE_Windows: seleniumLauncher('internet explorer', 'Windows 10', '11.103')
 
```

Comment:
BrowsersList (used by Babel) and SauceLabs have different definition of "latest". We'll use specific version number here and in .browserslistrc file.

It'd be good if we can derive one file from the other, but BrowsersList and SauceLabs use different formats to represent a browser. I don't know if that's worth the effort.